server/src/main/java/io/druid/query/lookup/LookupModule.java

```diff
@@ -146,24 +174,15 @@ public Object get(String id)
           }
 
           @Override
-          public Map<String, LookupExtractorFactory> getAll()
+          public LookupsState getAll()
```

Comment:
Instead of this returning lookup state, can it continue to return things that are viable to have for the POST requests?

I think the states for pending vs loading vs dropping can be exposed by a different endpoint like how the Overlord exposes the pending vs running vs waiting tasks.